lib/src/postgrest_filter_builder.dart

```diff
@@ -59,7 +63,11 @@ class PostgrestFilterBuilder extends PostgrestTransformBuilder {
   /// postgrest.from('users').select().neq('username', 'supabot')
   /// ```
   PostgrestFilterBuilder neq(String column, dynamic value) {
-    appendSearchParams(column, 'neq.$value');
+    if (value is List) {
+      appendSearchParams(column, 'eq.(${_cleanFilterArray(value)})');
```

Comment:
Should this be `neq`?
```suggestion
      appendSearchParams(column, 'neq.(${_cleanFilterArray(value)})');
```